src/kit.ts

```diff
@@ -663,13 +669,17 @@ export async function scanForKits(opt?: KitScanOptions) {
       scanPaths = scanPaths.concat(pathvar.split(sep));
     }
 
-    if (scanPaths) {
+    if (kit_options) {
```

Comment:
I don't think this branch will even be `false`, since `kit_options` is always a valid object